### ISSUE TITLE
bugs discord bot

### ISSUE THREAD

**Issue by oldpowerz** [Chalwk77/HALO-SCRIPT-PROJECTS#136]
![image](https://user-images.githubusercontent.com/71293715/155266827-c3d523b2-4d8e-43cf-80c2-f28fc1801e91.png)
in slayer lol 
and 
stack traceback:
        [string "Discord"]:902: in function <[string "Discord"]:897>
        [C]: in function '__index'
        [string "Discord"]:757: in function 'OnDeath'
        [string "Discord"]:848: in function <[string "Discord"]:847>
        EventCallback >> OnDeath >> Lua Error: [string "Discord"]:757: attempt to index local 'k' (a nil value)
        when try add second sv on bot

**Comment by Chalwk77** [comment]
Fixed.

Some minor changes were made to Discord.lua, settings.lua and main.lua

**Comment by oldpowerz** [comment]
new bug 

stack traceback:
        [string "Discord"]:902: in function <[string "Discord"]:897>
        [C]: at 0x73446090
        [C]: in function 'concat'
        [string "Discord"]:463: in function <[string "Discord"]:428>

**Comment by Chalwk77** [comment]
Yes, thanks. I was made aware of this bug by someone else as well.
It is just pure coincidence that the initial bug someone reported yesterday, once fixed, happened to cause a cascading barrage of further complications.

**Comment by Chalwk77** [comment]
Please use the latest version. The only file you will need is Discord.lua
